### PR TITLE
iosBuildStepsUpdate: project+workspace steps added

### PR DESCRIPTION
**Pull request by antonfelinsky**

### CODE REVIEW

File: Editor/Platforms/iOS/Post/Archive/Archive.cs [hunk 1]
```diff
@@ -11,20 +10,21 @@ namespace UniTools.Build
         fileName = nameof(Archive),
         menuName = MenuPaths.IOS + nameof(Archive)
     )]
-    public sealed class Archive : IosPostBuildStep
+    public class Archive : IosPostBuildStep
```
Comment (Rinal) [inline Editor/Platforms/iOS/Post/Archive/Archive.cs:13]: Must be abstract

File: Editor/Platforms/iOS/Post/Archive/Archive.cs [hunk 2]
```diff
@@ -11,20 +10,21 @@ namespace UniTools.Build
...
     {
         [SerializeField] private PathProperty m_projectPath = new PathProperty("Unity-iPhone.xcodeproj");
         [SerializeField] private PathProperty m_outputPath = new PathProperty("Unity-iPhone.xcarchive");
         [SerializeField] private string m_scheme = "Unity-iPhone";
         [SerializeField] private bool m_useModernBuildSystem = true;
+        protected virtual string CommandStart { get; set; }
```
Comment (Rinal) [inline Editor/Platforms/iOS/Post/Archive/Archive.cs:19]: Must be abstract

**Review by Rinal — CHANGES_REQUESTED** [review]
Please check PR comments
Update the package version in the current PR

**Comment by antonfelinsky** [comment]
@Rinal please, check updated PR

**Review by Rinal — CHANGES_REQUESTED** [review]
Please add a new package version

**Comment by antonfelinsky** [comment]
@Rinal package version updated

**Review by Rinal — APPROVED** [review]
Good job, ready for a merge

**Pull request merged**